package.json

```diff
@@ -183,7 +183,7 @@
     "jest": "^27.4.7",
     "kebab-case": "^1.0.1",
     "mocha": "^9.1.3",
-    "node-gyp": "8.4.1",
+    "node-gyp": "9.0.0",
```

Comment:
Same case here, if we are upgrading to v9 of node-gyp, we will want to also change all our CI scripts as well to reflect the same.

Reply:
will resolve based on #19